src/Calendar.jsx

```diff
@@ -390,7 +388,7 @@ let Calendar = React.createClass({
     let ToolbarToRender = components.toolbar || Toolbar
 
     return (
-      <div {...elementProps}
```

Comment:
This is a bit problematic. The user might be adding DOM events or `aria-`/`data-` attributes. Maybe in this case we add a new prop `elementProps` specifically for passing stuff to this div